webview/src/plots/components/App.test.tsx

```diff
@@ -134,32 +141,22 @@ describe('App', () => {
   } as TemplatePlotsData
 
   it('should send the initialized message on first render', () => {
-    render(<App />)
+    renderAppWithOptionalData()
     expect(mockPostMessage).toHaveBeenCalledWith({
       type: MessageFromWebviewType.INITIALIZED
     })
     expect(mockPostMessage).toHaveBeenCalledTimes(1)
   })
 
   it('should render the loading state when given no data', async () => {
-    render(<App />)
-    const loadingState = await screen.findByText('Loading Plots...')
-
-    expect(loadingState).toBeInTheDocument()
-  })
-
-  it('should render the loading state when not initially provided with collapsed sections', async () => {
```

Comment:
Removed because default data is now set in the webview and it's not a specific case anymore.